### DISCUSSION TITLE
프록시 서버란 무엇이며, 프록시서버가 필요한 이유가 무엇인가?

### DISCUSSION THREAD

**Discussion by Hyevvy** [prgrms-web-devcourse/We-are-the-MASTERS-OF-HTTP#52] [Interview Question]
프록시 서버란 무엇이고, 프록시 서버가 필요한 이유는 무엇인가요?

**Comment by sa02045** [comment]
프록시 서버는 클라이언트의 요청을 서버가 받기전에 중간서버역할을 하면서 다양한 기능을 수행하는 서버이다.

프록시 서버가 필요한 이유는 기능과 목적에 따라 프록시 서버를 두어 원래 서버가 여러 역할을 하는것을 분리할수 있으며 서버의 부하를 줄일 수 있다.

**Comment by punchdrunkard** [comment]
(의도한 답안) 포워드 프록시와 리버스 프록시로 나눠서 그 차이점을 위주로 설명해줬으면 좋겠다
